web/src/games/tictactoe/board.tsx

```diff
@@ -53,7 +53,7 @@ export class Board extends React.Component<IBoardProps, {}> {
   }
 
   _getGameOver() {
-    if (isOnlineGame(this.props.gameArgs) || isAIGame(this.props.gameArgs)) {
+    if ((isOnlineGame(this.props.gameArgs) || isAIGame(this.props.gameArgs)) && !isSpectator(this.props.playerID)) {
```

Comment:
Can this expression be turned into a helper method?  It's long and also duplicated here https://github.com/freeboardgames/FreeBoardGames.org/pull/676/files#diff-d09ac02e89c56265e00543c24377fb3a2849f155f794c688c40887111bd1e0caR39

Reply:
Yeah, I was thinking the same thing. But I don't know how to describe it. I think it should be "isFirstPersonView", maybe ? Because the other methods are watching the game as a "third-person". Do you have any ideas ?

Reply:
It is also duplicated in other games that support all modes (chess, for instance)